rotkehlchen/api/rest.py

```diff
@@ -3787,13 +3787,17 @@ def get_nfts_balances(self, async_query: bool, ignore_cache: bool) -> Response:
         result_dict = {'result': response['result'], 'message': response['message']}
         return api_response(process_result(result_dict), status_code=status_code)
 
-    def get_nfts_with_price(self) -> Response:
-        return self._api_query_for_eth_module(
-            async_query=False,
-            module_name='nfts',
-            method='get_nfts_with_price',
-            query_specific_balances_before=None,
-        )
+    def get_manual_current_prices(self) -> Response:
```

Comment:
Shouldn't this be a paginated one? Perhaps an overkill for now but in the future I can see this becoming too long. Can also go in another PR to unblock Luki faster.

Reply:
I asked luki about pagination in this endpoint and told me to keep it unpaginated. Should I change it? I also wonder the same

Reply:
if you want to make it paginated, I need backend to support filter by asset too 😛

Reply:
I mean this is easy I guess. The point is to be future-proof. How should this endpoint be for the frontend? @kelsos ?

Reply:
I am not sure how pagination would help in this cache, for future-proofing we would need to change the approach we work with prices in the frontend.

Currently, the prices for owned assets are also cached in the state.

Reply:
okay but the idea is to go towards pagination everything and not cache stuff in the frontend, right?